src/CommunityVoices/Public/js/batch-upload-confirm.js

```diff
@@ -22,7 +22,14 @@ function postData(form) {
         $.ajax({
           url : $("#actualForm").attr('action'),
           type: $("#actualForm").attr('method'),
-          data: data
+          data: data,
+          success: function () {
+            // This goes back to /quotes because we are at /quotes/confirm
+            window.location.href = '.';
```

Comment:
@Sammidysam so right now even if the user just uploads one source and it's quotes, it will redirect. (Note that this post request is called when either all sources/quotes submitted or a single source. Even if all sources are submitted we may not want to redirect as I am not uploading any sources that have issues or it's associated quotes have issues. My thinking is that we want to give the user an ability to fix any errors and should only redirect when all of their sources have been uploaded, or all of their sources remaining have no associated quotes (this will only happen if the user makes a mistake on the sheet and creates a source identifier that is unpaired). 

This is a little confusing I realize, but does this make sense?

Reply:
Thanks so much for finding this.  I did indeed miss the procedure of submitting only one quote at a given time.  I'll add the logic you mention.

Reply:
Fixed.

Reply:
Although I didn't redirect when they have sources with no quotes, as I am thinking they can stare at that and get confused and move on from there.